src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -112,14 +113,18 @@ public void Log(LogLevel logLevel, int eventId, object state, Exception exceptio
             }
             else
             {
-                message = LogFormatter.Formatter(state, exception);
+                message = state?.ToString();
```

Comment:
POI - is this acceptable?
Need a default if formatter is null and since we removed LogFormatter